src/resources/chain.rs

```diff
@@ -280,6 +283,232 @@ impl<T: Block> Chain<T> {
         Err(unsafe { OwnedChain::from_vec_unchecked(res) })
     }
 
+    /// Returns an OwnedChain<T> containing the difference, i.e. the
```

Comment:
The summary of the doc string should be one line only. I think just ‘Returns a chain with the items that are in `self` but not in `other`’ will do.